### PR TITLE
Fix to constraining `jinja2 <3` to `markupsafe <2`

### PR DESCRIPTION
**Pull request by xylar**
These were missed in
https://github.com/conda-forge/conda-forge-repodata-patches-feedstock/pull/238

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* Bumped the build number (if the version is unchanged)
* Reset the build number to `0` (if the version changed)
* [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* Ensured the license file is being packaged.

### CODE REVIEW

**Comment by xylar** [comment]
Here is the output of `./show_diff.py`, indicating that several more packages were, indeed, caught:
```
$ ./show_diff.py 
Downloading: https://conda.anaconda.org/conda-forge/noarch/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/noarch/repodata.json.bz2
noarch::jinja2-2.10-py_1.tar.bz2
-    "markupsafe >=0.23",
+    "markupsafe >=0.23,<2",
Downloading: https://conda.anaconda.org/conda-forge/linux-64/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/linux-64/repodata.json.bz2
linux-64::jinja2-2.10-py27_0.tar.bz2
-  "constrains": [
-    "python_abi * *_cp27mu"
-  ],
-    "markupsafe >=0.23",
+    "markupsafe >=0.23,<2",
-  "version": "2.10"
+  "version": "2.10",
+  "constrains": [
+    "python_abi * *_cp27mu"
+  ]
linux-64::jinja2-2.10-py35_0.tar.bz2
-  "constrains": [
-    "python_abi * *_cp35m"
-  ],
-    "markupsafe >=0.23",
+    "markupsafe >=0.23,<2",
-  "version": "2.10"
+  "version": "2.10",
+  "constrains": [
+    "python_abi * *_cp35m"
+  ]
linux-64::jinja2-2.10-py36_0.tar.bz2
-  "constrains": [
-    "python_abi * *_cp36m"
-  ],
-    "markupsafe >=0.23",
+    "markupsafe >=0.23,<2",
-  "version": "2.10"
+  "version": "2.10",
+  "constrains": [
+    "python_abi * *_cp36m"
+  ]
Downloading: https://conda.anaconda.org/conda-forge/linux-armv7l/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/linux-armv7l/repodata.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/linux-aarch64/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/linux-aarch64/repodata.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/linux-ppc64le/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/linux-ppc64le/repodata.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/osx-64/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/osx-64/repodata.json.bz2
osx-64::jinja2-2.10-py27_0.tar.bz2
-  "constrains": [
-    "python_abi * *_cp27m"
-  ],
-    "markupsafe >=0.23",
+    "markupsafe >=0.23,<2",
-  "version": "2.10"
+  "version": "2.10",
+  "constrains": [
+    "python_abi * *_cp27m"
+  ]
osx-64::jinja2-2.10-py35_0.tar.bz2
-  "constrains": [
-    "python_abi * *_cp35m"
-  ],
-    "markupsafe >=0.23",
+    "markupsafe >=0.23,<2",
-  "version": "2.10"
+  "version": "2.10",
+  "constrains": [
+    "python_abi * *_cp35m"
+  ]
osx-64::jinja2-2.10-py36_0.tar.bz2
-  "constrains": [
-    "python_abi * *_cp36m"
-  ],
-    "markupsafe >=0.23",
+    "markupsafe >=0.23,<2",
-  "version": "2.10"
+  "version": "2.10",
+  "constrains": [
+    "python_abi * *_cp36m"
+  ]
Downloading: https://conda.anaconda.org/conda-forge/osx-arm64/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/osx-arm64/repodata.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/win-32/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/win-32/repodata.json.bz2
win-32::jinja2-2.10-py27_0.tar.bz2
-  "constrains": [
-    "python_abi * *_cp27m"
-  ],
-    "markupsafe >=0.23",
+    "markupsafe >=0.23,<2",
-  "version": "2.10"
+  "version": "2.10",
+  "constrains": [
+    "python_abi * *_cp27m"
+  ]
win-32::jinja2-2.10-py35_0.tar.bz2
-  "constrains": [
-    "python_abi * *_cp35m"
-  ],
-    "markupsafe >=0.23",
+    "markupsafe >=0.23,<2",
-  "version": "2.10"
+  "version": "2.10",
+  "constrains": [
+    "python_abi * *_cp35m"
+  ]
win-32::jinja2-2.10-py36_0.tar.bz2
-  "constrains": [
-    "python_abi * *_cp36m"
-  ],
-    "markupsafe >=0.23",
+    "markupsafe >=0.23,<2",
-  "version": "2.10"
+  "version": "2.10",
+  "constrains": [
+    "python_abi * *_cp36m"
+  ]
Downloading: https://conda.anaconda.org/conda-forge/win-64/repodata_from_packages.json.bz2
Downloading: https://conda.anaconda.org/conda-forge/win-64/repodata.json.bz2
win-64::jinja2-2.10-py27_0.tar.bz2
-  "constrains": [
-    "python_abi * *_cp27m"
-  ],
-    "markupsafe >=0.23",
+    "markupsafe >=0.23,<2",
-  "version": "2.10"
+  "version": "2.10",
+  "constrains": [
+    "python_abi * *_cp27m"
+  ]
win-64::jinja2-2.10-py35_0.tar.bz2
-  "constrains": [
-    "python_abi * *_cp35m"
-  ],
-    "markupsafe >=0.23",
+    "markupsafe >=0.23,<2",
-  "version": "2.10"
+  "version": "2.10",
+  "constrains": [
+    "python_abi * *_cp35m"
+  ]
win-64::jinja2-2.10-py36_0.tar.bz2
-  "constrains": [
-    "python_abi * *_cp36m"
-  ],
-    "markupsafe >=0.23",
+    "markupsafe >=0.23,<2",
-  "version": "2.10"
+  "version": "2.10",
+  "constrains": [
+    "python_abi * *_cp36m"
+  ]
```

**Comment by xylar** [comment]
@jakirkham, since you reviewed #238, could you have a look at this, too?

**Comment by xylar** [comment]
@conda-forge/core, this is ready to review.  Thanks!

File: recipe/gen_patch_json.py [hunk 1]
```diff
@@ -1203,8 +1203,9 @@ def _gen_new_index_per_key(repodata, subdir, index_key):
         # incompatibilities with markupsafe >=2.1 and we are constraining
         # markupsafe <2 to be on the safe side
         # https://github.com/pallets/jinja/issues/1585
-        if record_name == "jinja2" and record['version'].startswith(
-                ('2.9.', '2.10.', '2.11.')):
+        if record_name == "jinja2" and \
+                (record['version'].startswith(('2.9.', '2.10.', '2.11.')) or
```
Comment (hmaarrfk) [inline recipe/gen_patch_json.py:1207]: can you be more precise about the build number for 2.11.3? You seem to be building it again, so we don't want new builds to get patched again.

Comment (xylar) [reply recipe/gen_patch_json.py:1207]: Sure, that's a good idea.

Comment (xylar) [reply recipe/gen_patch_json.py]: @hmaarrfk, I med your suggested change.  It didn't change the output of `show_diff.py` as far as I can tell but I pasted the new output above in the PR.

**Comment by xylar** [comment]
Thanks, @hmaarrfk!

**Pull request merged**